packages/protocol/contracts/L1/TaikoL1.sol

```diff
@@ -247,7 +247,10 @@ contract TaikoL1 is EssentialContract, IHeaderSync, TaikoEvents {
     }
 
     function getLatestSyncedHeader() public view override returns (bytes32) {
-        return state.getL2BlockHash(state.latestVerifiedHeight);
+        return
+            state.getL2BlockHash(
```

Comment:
https://github.com/taikoxyz/taiko-mono/pull/13042

@cyberhorsey I think we need the fix as above.